service/role.go

```diff
@@ -339,6 +364,19 @@ func (r *roleService) updateRoleToken(ctx context.Context, domain, role, proxyFo
 	return rt.(*RoleToken), err
 }
 
+// TODO:
+func roleCacheMemoryUsage(data *cacheData) int64 {
+	tokenSize := int64(unsafe.Sizeof(data.token)) + int64(len(data.token.Token))
+	expiryTimeSize := int64(unsafe.Sizeof(data.token.ExpiryTime)) + int64(len(strconv.FormatInt(data.token.ExpiryTime, 10)))
```

Comment:
do not need to convert to string?
```suggestion
	expiryTimeSize := int64(unsafe.Sizeof(data.token.ExpiryTime))
```